.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+
+{
```

Comment:
should this be checked in?

Reply:
This is needed for debuggability purpose. (To follow the debugging process from visual studio)
All the other repos had one and I added it here too so that its easier for a developer to debug